### PR TITLE
Fixed transparent input backgrounds and disabled state borders

### LINKED ISSUE #568: Background can be seen through select and searchable select
### Describe the Bug

Background can be seen through select and searchable select

image
image

This is even more obvious in web modeler

image

### Steps to Reproduce

1. add two selects to a form
2. configure one as searchable
3. select them in the editor

### Expected Behavior

The blue background should not be visible

### PR DESCRIPTION
**Pull request by Skaiir**
Closes #568

image

### CODE REVIEW

**Review by vsgoulart — CHANGES_REQUESTED** [review]

File: packages/form-js-viewer/src/render/components/form-fields/Number.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ export default function Numberfield(props) {
           autoComplete="off"
           step={ arrowIncrementValue }
           value={ displayValue } />
-        <div class={ classNames('fjs-number-arrow-container', { 'fjs-disabled': disabled }) }>
```
[the remark this answers is not in the document]
Comment (Skaiir) [reply packages/form-js-viewer/src/render/components/form-fields/Number.js:190]: I did not, fair enough, won't touch them then.

**Comment by pinussilvestrus** [comment]
If we merge this to `master` (which makes sense), I'm wondering whether auto-merge to `develop` will work here as we are touching files that do not exist on `develop` anymore (they were renamed to `*-base.css`) 🤔

**Comment by Skaiir** [comment]
@pinussilvestrus I guess it won't, but then we have to manually do it instead I guess? Or I can hold off on merging this one entirely.

**Review by vsgoulart — CHANGES_REQUESTED** [review]
image
It's still broken to me. Is the deployment up to date?

**Comment by Skaiir** [comment]
@vsgoulart It should be, but maybe there's some issues with stylesheets? Not sure why the changes aren't on there, could you try locally?

**Review by vsgoulart — APPROVED** [review]

**Comment by pinussilvestrus** [comment]
> @vsgoulart ~It should be, but maybe there's some issues with stylesheets? Not sure why the changes aren't on there, could you try locally?~
> 
> Nevermind new deployment got you covered, took a bit to update the cache.

Yeah it can took a while until the update is done, unfortunately Netlify build hooks are async.

**Pull request merged**